Formula/geogig.rb

```diff
@@ -0,0 +1,22 @@
+class Geogig < Formula
+  desc "Enable distributed versioning of geospatial data in a git-like context"
+  homepage "http://geogig.org"
+  url "http://download.locationtech.org/geogig/geogig-1.2.0.zip"
+  sha256 "c438d97d8978a9d4985bd4f2ba3fff481b7956bfc4bd986ee3c79c6c07cd4f34"
+
+  bottle :unneeded
+
+  depends_on :java => ["1.8+"]
+
+  def install
+    sbin.install "bin/geogig"
```

Comment:
Don't put the same executable in `sbin` and `bin`. You'll end up with one that behaves correctly and one that borks because it's not inside the Java wrapper.